builtin/providers/pass/resource_pass_password.go

```diff
@@ -0,0 +1,78 @@
+package pass
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func passPasswordResource() *schema.Resource {
+	return &schema.Resource{
+		Create: passPasswordResourceWrite,
+		Update: passPasswordResourceWrite,
+		Delete: passPasswordResourceDelete,
+		Read:   passPasswordResourceRead,
+
+		Schema: map[string]*schema.Schema{
+			"path": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Full path where the pass password will be written.",
+			},
+
+			// Data is passed as JSON so that an arbitrary structure is
+			// possible, rather than forcing e.g. all values to be strings.
+			"data": &schema.Schema{
```

Comment:
I'm not sur this is required, as if the password can't be unmarshaled, we fallback to plain text.